reactquiz/src/components/quiz/Results.js

```diff
@@ -1,34 +1,25 @@
-import React,{Component} from 'react';
-
-
-class Results extends Component{
-
+import React, { Component } from 'react';
 
+class Results extends Component {
 	render() {
-
-		var percent =(this.props.score / this.props.questions.length*100);
-		if (percent > 80){
-
-			var message ='Awesome Job!';
-		} else if(percent < 80 && percent >60)
-		{
-			var message = 'You did Ok !';
-		}
-		else {
-
-			var message ='Not satisfactory. Should try once again !';
+		const percent = (this.props.score / this.props.questions.length * 100 * 100);
+		let message;
+		if (percent > 80) {
+			message = 'Awesome Job!!';
+		} else if (percent > 60) {
+			message = 'You did Ok!';
+		} else {
+			message = 'Not satisfactory. Should try once again!!!';
```

Comment:
Minor suggestion: You have some inconsistent spacing and capitalization in your messages ("Awesome Job!!", "You did Ok!", "Not satisfactory. Should try once again!!!"). Consider making them more uniform for better readability.